vm-mock/vm.js

```diff
@@ -454,8 +485,12 @@ export default function createMockedABI(
         scCallMockStack.push(getArrayBuffer(ptr));
       },
 
-      assembly_script_call(_address, method, _param, _coins) {
+      assembly_script_call(_address, method, _param, coins) {
         if (scCallMockStack.length) {
+          if(BigInt(coins) > getCalleeBalance()) {
+            ERROR('Not enough balance to pay the call to ' + ptrToString(method));
+          }
+          // todo: remove spent coins from the callee balance
```

Comment:
TODO

Reply:
yes. i propose to let this topic for a futur task

Reply:
done